src/chapters/strings/encoding-characters.rst

```diff
@@ -0,0 +1,119 @@
+
+Representing Numbers
+--------------------
+
+.. index:: ! byte
+
+.. index::
+   pair: number; binary
+   pair: number; decimal
+
+A **byte** is a set of 8 bits. Since each bit can have one of two values, each byte can have 2\ :sup:`8` = 256 different values. These look like 00101101 or 11110011, and they represent a **binary number**, or a base-2 number. A binary number is a number representatation that uses only 0s and 1s. The numbers that you are used to using---which are built out of integers 0...9---are **decimal numbers**, or base-10 numbers.
+
+It may not be obvious, but every decimal integer can be represented as a binary integer, and vice versa. There are 256 different values a byte may take, each of which can be used to represent an integer, from 0 to 255. 
+
+.. note:: We will not discuss binary to decimal number conversion. If you are interested in learning more, there are `many <https://www.csetutor.com/how-to-convert-binary-to-decimal-examples/>`_ `tutorials <https://www.youtube.com/watch?v=wPvI19DmWQw>`_ `online <https://www.khanacademy.org/math/algebra-home/alg-intro-to-algebra/algebra-alternate-number-bases/v/decimal-to-binary>`_.
+
+In this way, the bits in a computer can be viewed as integers. If you want to represent values greater than 255, just use more bits! 
+
+Representing Strings
+--------------------
+
+Strings are collections of characters, so if we can represent each character as a number, then we'll have a way to go from a string to a collection of bits, and back again.
+
+Character Encodings
+^^^^^^^^^^^^^^^^^^^
+
+.. index::
+   pair: character; encoding
+
+Unlike the natrual translation between binary and decimal numbers, there is no natural translation between integers and characters. For example, you might create a pairing of 0 to ``a``, 1 to ``b``, and so on. But what integer should be paired with ``$`` or a tab? Since there is no natural way to translate between characters and integers, computer scientists have had to make such translations up. Such translations are called **character encodings**.
+
+.. index:: Unicode
+
+There are many different encodings, some of which continue to evolve as the way in which we use data evolves. For instance, the most recent versions of the Unicode standard include emoji characters, such as 🌮.
+
+The ASCII Encoding
+^^^^^^^^^^^^^^^^^^
+
+.. index:: ASCII
+
+Most of the characters that you are used to using---including letters, numbers, whitespace, punctuation, and symbols---are part of the **ASCII** (pronounced *ASS-kee*) character encoding. This standard has changed very little since the 1960s, and is a subset of all other commonly-used encodings. In other words, other encodings start with ASCII and built up to include other characters.
```

Comment:
Typo: No comma needed after 1960s.

Alternative phrasing: "This standard has changed very little since the 1960s, and it is the foundation of all other commonly-used encodings."